starcounter-include.html

```diff
@@ -249,24 +249,30 @@
         StarcounterIncludePrototype._notifyPath = function(path, value) {
             if (path.indexOf("partial.") === 0 || path.indexOf("viewModel.") === 0) {
                 if (
-                    path === "partial.Html" ||
-                    // path.match(/^partial\.[^.]+\.Html$/)
-                    (path.indexOf('.Html') === path.length - 5) && path.slice(8, -5).indexOf('.') === -1
+                    // path === "viewModel.Html" || // covered by  viewModel._whatever_
+                    // path.match(/^viewModel\.[^.]+\.Html$/)
+                    (path.indexOf('.Html') === path.length - 5) && path.slice(10, -5).indexOf('.') === -1
                 ) {
                     this.partialChanged(this.partial);
                 } else if (
-                    path === "viewModel.Html" ||
-                    // path.match(/^viewModel\.[^.]+\.Html$/)
-                    (path.indexOf('.Html') === path.length - 5) && path.slice(10, -5).indexOf('.') === -1
+                    // path === "partial.Html" || // covered by  partial._whatever_
+                    // path.match(/^partial\.[^.]+\.Html$/)
+                    (path.indexOf('.Html') === path.length - 5) && path.slice(8, -5).indexOf('.') === -1
                 ) {
                     this.partialChanged(this.partial);
+                } else
+
+                // if viewModel|partial._whatever_, but not _ver - scope changed
+                if(path.split('.').length === 2 && path.indexOf('_ver#s') === -1 && path.indexOf('_ver#c$') === -1){
+                    this.partialChanged(this.partial);
```

Comment:
> The common way is:

If you say so. I'll change, however personally I find it not so visible.

> But then the remainder of the condition is always true, isn't it? && path.indexOf('_ver#s') === -1 && path.indexOf('_ver#c$') === -1.

It's not for: `viewModel._ver#s`. Like in:
```html
<template is="dom-bind">
    <starcounter-include view-model="{{model}}"></starcounter-include>
</template>
<script>
    let palindrom_obj = {
        _ver#s: 0,
        _ver#c$: 0,
        smth: {}
    };
    document.querySelector('[is="dom-bind"]').set('model', palindrom_obj);
    // receive change from the server
    document.querySelector('[is="dom-bind"]').set('model._ver#s', 1);
</script>
```